packages/notifi-react-card/lib/components/subscription/EventTypeHealthCheckRow.tsx

```diff
@@ -0,0 +1,205 @@
+        parseFloat(customValue.slice(0, -1)) >= 1 &&
+        parseFloat(customValue.slice(0, -1)) <= 99
+      ) {
+        console.log(selectedRatio);
+        return;
+        //TODO: hook up API call here
+        // if (!enabled) {
+        //   instantSubscribe({
+        //     alertConfiguration: alertConfiguration,
+        //     alertName: alertName,
+        //   });
+        // } else {
+        //   instantSubscribe({
+        //     alertConfiguration: null,
+        //     alertName: alertName,
+        //   });
+        // }
+      } else {
+        setErrorMessage('Please enter a valid number');
+      }
+    }
+  };
+
+  return (
+    <div>
+      <div
+        className={clsx(
+          'EventTypeHealthCheckRow__container',
+          classNames?.container,
+        )}
+      >
+        <div
+          className={clsx('EventTypeHealthCheckRow__label', classNames?.label)}
+        >
+          {config.name}
+          {tooltipContent !== undefined && tooltipContent.length > 0 ? (
+            <NotifiTooltip
+              classNames={classNames?.tooltip}
+              content={tooltipContent}
+            />
+          ) : null}
+        </div>
+        <NotifiToggle
+          checked={enabled}
+          classNames={classNames?.toggle}
+          disabled={disabled}
+          setChecked={handleNewSubscription}
+        />
+      </div>
+      {isShowSelectButton ? (
+        <>
+          <div
+            className={clsx(
+              'EventTypeHealthCheckRow__content',
+              classNames?.content,
+            )}
+          >
+            Alert me when my margin ratio is {ratios[0]?.type}
+          </div>
+          <div
+            className={clsx(
+              'EventTypeHealthCheckRow__buttonContainer',
+              classNames?.buttonContainer,
+            )}
+          >
+            {ratios.map((value, index) => {
+              const percentage = value.ratio * 100 + '%';
+              return (
+                <div
+                  key={index}
+                  className={clsx(
+                    'EventTypeHealthCheckRow__button',
+                    `${
+                      index === selectedIndex
+                        ? ' EventTypeHealthCheckRow__buttonSelected'
+                        : undefined
+                    }`,
+                    classNames?.button,
+                  )}
+                  onClick={() => {
+                    setSelectedIndex(index);
+                    setCustomButtonPlaceholder('Custom');
+                    setCustomValue('');
+                    setSelectedRatio(percentage);
+                    setErrorMessage('');
+                  }}
+                >
+                  {percentage}
+                </div>
+              );
+            })}
+            <input
+              onClick={() => {
+                //assume we should only allow no more than two specific values and one custom value here
```

Comment:
Agreed here. I also was thinking if a tenant would allow a user to add a custom value and if that option should be configured in the config tool.

Reply:
I'll confirm with Sam and Juni and will address this in next PR(connect API)

Reply:
asked Juni, we should limited the check ratios no more than two from the admin config when tenant set up. I'll open a PR to address it in admin tool

Reply:
Thank you for checking!